docker/minder/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: Copyright 2023 The Minder Authors
 # SPDX-License-Identifier: Apache-2.0
 
-FROM index.docker.io/library/golang:1.23.4@sha256:574185e5c6b9d09873f455a7c205ea0514bfd99738c5dc7750196403a44ed4b7 AS builder
+FROM index.docker.io/library/golang:1.23.6@sha256:adfbe17b774398cb090ad257afd692f2b7e0e7aaa8ef0110a48f0a775e3964f4 AS builder
```

Comment:
Do we need this Dockerfile at all?

Reply:
It's part of a test leg... I think some folks were using `docker compose` to kick off local Dockerfile builds (vs using `ko`'s cross-build capability).

I'd be happy to ditch it, maybe I'll ask on the #minder OpenSSF channel.

Reply:
I think we can ditch it